pandas/tests/generic/test_series.py

```diff
@@ -157,3 +157,8 @@ def finalize(self, other, method=None, **kwargs):
             result = pd.concat([ser, ser2])
             assert result.filename == "foo+bar"
             assert result.name is None
+
+    def test_series_with_complex_nan(self):
+        # GH#53627
+        ser = Series([complex("nan")])
```

Comment:
can you also check `ser = Series([complex('1+j'), complex('nan'), complex('2+2j')])`?  thats the actual case that i first stumbled on